### DISCUSSION TITLE
MAP BASE / TEMPLATE

### DISCUSSION THREAD

**Discussion by Lunaire333** [AmyE123/TactiKit#8] [Map Editor Ideas]
It would be great if the user could have the option of a map base or a template to start designing a map, mainly because not everyone is familiar with or has the inspiration for a map concept/design.

So, having the option to start from a base would give more inspiration to the user.

![image](https://github.com/AmyE123/TactiKit/assets/158181584/a75e6247-27bc-468c-b9a4-a971ac29690b)

**Comment by AmyE123** [comment]
On the load button, it comes up with examples which you can load in (this isn't populated yet). Do you think I should move that here, or have it in both the load and create new popups?

**Reply by bLInVoid** [reply]
I would say have it in just the create new popup, if you use a template /base then it will create a new map  anyway, I don't think it would make sense to have it in load since that section is usually only for maps the user has intentionally created, it could also confuse a beginner by having maps they don't remember making in the load popup.

**Reply by Lunaire333** [reply]
as @bLInVoid said so well, it would be better to just have those map base/templates in the "create" new popup. Would be less confusing and understandable for any user (from beginner to expert).
